### ISSUE TITLE
Bot doesn't play any music, no errors in console

### ISSUE THREAD

**Issue by agraubert** [Just-Some-Bots/MusicBot#1384]
The music bot is silently failing to play any songs. After a `!play` command, it downloads the song, but it's never queued or played (no sound and `!queue` or `!np` report that there are no songs playing).

 ```
Connected!  Musicbot v1.9.5_2                                                                                                                           

Bot:   <ID>/Tenno DJ#9082
Owner: <ID>/Jroot#8988

Server List:
 - Gaming chat

Bound to text channels:
 - Gaming chat/jukebox
 - Gaming chat/testing_grounds

Autojoining voice chanels:
 - Gaming chat/Testing Grounds

Options:
  Command prefix: !
  Default volume: 15%
  Skip threshold: 4 votes or 50%
  Now Playing @mentions: Disabled
  Auto-Summon: Disabled
  Auto-Playlist: Disabled
  Auto-Pause: Enabled
  Delete Messages: Enabled
    Delete Invoking: Enabled
  Debug Mode: Disabled
  Downloaded songs will be deleted

Deleting old audio cache
Attempting to autojoin Testing Grounds in Gaming chat
Attempting connection...
Connection established.

[Command] <ID>/Jroot (!play she swallowed burning coals)
[Download] Started: https://www.youtube.com/watch?v=c3UGNFUIiV4
[Download] Complete: https://www.youtube.com/watch?v=c3UGNFUIiV4
[Command] <ID>/Jroot (!np)
<later>
[Command] <ID>/Jroot (!shutdown)
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7fa77f592320>
```

It has worked fine in the past, and this issue is pretty recent, which makes me think it's an issue with a python or system dependency.  I've checked the requirements.txt file and I have everything listed installed.
If you have any ideas about this, I'd really appreciate the help.

**Comment by SeiZaNex** [comment]
Have same issue since this afternoon.

The bot seems to have downloaded the file and then ignores it completely. As if he doesn't want to read it.

I tried updating and reinstalling the dependencies. Nothing seems to have worked.

I'm out of ideas.

**Comment by SeiZaNex** [comment]
I've tried to run through all the files / code. 
I've just figured that the files the bot tries to read are "empty stream" for some reason. 
My guess is that youtube-dl isn't working the same way it used to. Any other leads?

**Comment by jayktaylor** [comment]
Not able to reproduce this. Can you try updating the bot and running it again?

**Comment by katiras** [comment]
I have the same problem. When I drag n drop BOT through channels some people start listening to it.
Can you guys try this to see if its the same issue?

**Comment by jayktaylor** [comment]
If this isn't resolved in 1.9.6_1, please create another issue.

**Comment by thanoskat** [comment]
It happened again with the new version. So it is not solved yet. Although I cannot reproduce it on demand.
